src/components/Chat/Header.js

```diff
@@ -157,7 +158,7 @@ const styles = StyleSheet.create({
   name: {
     marginLeft: 11,
     fontFamily: fonts.inter[600],
-    color: '#fff',
+    color: COLORS.white,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good job replacing the hardcoded color value with a dynamic one from the theme object. This improves maintainability and consistency of the code.
```diff
- color: '#fff',
+ color: COLORS.white,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->